examples/c++/vector-api/README.md

```diff
@@ -0,0 +1,30 @@
+# Icicle Example: Vector Operations API
+
+## Key-Takeaway
+
+The Vector Operations API supports the following:
+
+ - element-wise vector operations (e.g. addition, multiplication)
+ - vector reduction operations (e.g. sum of elements, product of elements)
+ - scalar-vector operations (e.g add scalar to vector)
+ - matrix operations (e.g. transposition)
+ - miscellaneous operations like bit-reversal and slicing. 
+ 
+ All these operations can be performed on a host or device both synchronously and asynchronously.
+
+## Running the example
+
+```sh
+# for CPU
+./run.sh -d CPU
+# for CUDA
+./run.sh -d CUDA -b /path/to/cuda/backend/install/dir
```

Comment:
this won't work right? I mean since it's not implemented for CUDA

Reply:
no CUDA yet. I removed CUDA in README. I will add later, when we have it in vector ops.